sdk/network/azure-mgmt-network/azure/mgmt/network/_client.py

```diff
@@ -261,6 +261,8 @@ def __init__(
             api_version=api_version,
             profile=profile
         )
+        self._serialize = Serializer(self._models_dict())
+        self._deserialize = Deserializer(self._models_dict())
```

Comment:
thanks so much for fixing! This looks good, can you also add `self._serialize.client_side_validation = False`? Thanks! https://github.com/Azure/autorest.python/blob/df3f272f18e5f2f2c69933e72a417eae14be7b3e/packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/Lro/lro/_auto_rest_long_running_operation_test_service.py#L49-L51